src/components/dialogmote/innkalling/virksomhet/VirksomhetRadioGruppe.tsx

```diff
@@ -36,8 +36,20 @@ export const VirksomhetRadioGruppe = ({
     velgVirksomhet("");
   };
 
+  let defaultVirksomhet: string | undefined = undefined;
+  if (virksomheter.length === 1) {
+    defaultVirksomhet = virksomheter[0];
+    removeInputAndChooseVirksomhet(defaultVirksomhet);
+  }
+
   return (
-    <RadioGroup legend={label} size="small" id={id} error={error}>
+    <RadioGroup
+      legend={label}
+      size="small"
+      id={id}
+      error={error}
+      defaultValue={defaultVirksomhet}
+    >
```

Comment:
`removeInputAndChooseVirksomhet` setter verdien i formet, og `defaultValue={defaultVirksomhet}` huker av radioknappen. Så vidt jeg kan se, må begge gjøres for at det både skal se riktig ut og faktisk fungere 👍